pom.xml

```diff
@@ -29,7 +29,20 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-math3</artifactId>
       <version>3.6.1</version>
-    </dependency>  
+    </dependency>
+      <dependency>
+          <groupId>org.junit.jupiter</groupId>
```

Comment:
Why do we need this dependency? The `junit` above is not enough?

Reply:
When I cloned under IntelliJ, there was only one JUnit dependency for 4.x.x in POM but no tests run. JUnit 5.x.x were added, everything was fine. I out-commented the 4.x.x but no tests run so decided to leave both, if that was your question about.